core/rawdb/accessors_indexes.go

```diff
@@ -129,3 +130,30 @@ func WriteBloomBits(db ethdb.KeyValueWriter, bit uint, section uint64, head comm
 		log.Crit("Failed to store bloom bits", "err", err)
 	}
 }
+
+// DeleteBloombits removes all compressed bloom bits vector belonging to the
+// given section range and bit index.
+func DeleteAllBloombits(db ethdb.Database, bit uint, from uint64, to uint64) {
+	key := append(append(bloomBitsPrefix, make([]byte, 2)...))
+	binary.BigEndian.PutUint16(key[1:], uint16(bit))
+
+	it := db.NewIteratorWithPrefix(key)
+	defer it.Release()
+
+	for it.Next() {
+		if len(it.Key()) != len(bloomBitsPrefix)+2+8+32 {
```

Comment:
Just like in the canonical hash iterator, you should check for `bloomBitsPrefix` here.